### PR TITLE
Pyodide headers

### PR DESCRIPTION
**Pull request by sra405**
closes https://github.com/RaspberryPiFoundation/digital-editor-issues/issues/303

### CODE REVIEW

File: webpack.component.config.js [hunk 1]
```diff
@@ -88,6 +95,25 @@ module.exports = {
       "Access-Control-Allow-Methods": "GET, POST, PUT, DELETE, PATCH, OPTIONS",
       "Access-Control-Allow-Headers":
         "X-Requested-With, content-type, Authorization",
+      // Pyodide - required for input and code interruption - needed on the host app
+      "Cross-Origin-Opener-Policy": "same-origin",
+      "Cross-Origin-Embedder-Policy": "require-corp",
```
Comment (loiswells97) [inline webpack.component.config.js:100]: So are we just assuming that host pages will have to add these themselves for now, or am I not understanding?

**Review by loiswells97 — COMMENTED** [review]
Just a few questions, but otherwise woohoo!

File: src/PyodideWorker.js [hunk 2]
```diff
@@ -164,7 +166,7 @@ const PyodideWorker = () => {
       before: async () => {
         pyodide.registerJsModule("basthon", fakeBasthonPackage);
         await pyodide.loadPackage(
-          `${process.env.ASSETS_URL}/packages/turtle-0.0.1-py3-none-any.whl`,
+          `${process.env.ASSETS_URL}/pyodide/packages/turtle-0.0.1-py3-none-any.whl`,
```
[the remark this answers is not in the document]
Comment (sra405) [reply src/PyodideWorker.js:169]: ah they can both probably be merged into one env var now. When the site was deployed and assets were in the bucket but on the same domain it was causing a problem so I exposed the bucket via a URL and hit that directly a while back. But now its all bucket based

File: src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx [hunk 3]
```diff
@@ -134,8 +130,6 @@ const PyodideRunner = ({ active }) => {
   };
 
   const handleInput = async () => {
-    // TODO: Sk.sense_hat.mz_criteria.noInputEvents = false;
-
```
[the remark this answers is not in the document]
Comment (sra405) [reply src/components/Editor/Runners/PythonRunner/PyodideRunner/PyodideRunner.jsx:138]: copilot 🤦 will restore

File: webpack.component.config.js (hunk 1 above)
Comment (sra405) [reply webpack.component.config.js:100]: yea that's right. So on editor.raspberrypi.org we've already got these set 👍

File: src/PyodideWorker.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (sra405) [reply src/PyodideWorker.js:169]: changing them all to `ASSETS_URL` in this file then we can remove that later if we want to 👍

**Review by loiswells97 — APPROVED** [review]
WOOOOHOOOOO 🥳 🎉 🚢 🚀

**Pull request merged**